ctk/idp/src/test/kotlin/org/codice/compliance/web/slo/ServiceProviderEndpointSpec.kt

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compliance.web.slo
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.StringSpec
+import org.codice.compliance.web.slo.endpoint.ServiceProviderEndpoint
+import org.codice.compliance.web.slo.endpoint.ServiceProviderEndpoint.Companion.BACK_CHANNEL
+import java.io.File
+
+class ServiceProviderEndpointSpec : StringSpec() {
```

Comment:
❗️  Should this also test that hitting the endpoints writes true/false to the file as well?

Reply:
The work on the second SP isn't complete, there may be changes and more tests